cpp/src/arrow/util/config.h.cmake

```
@@ -59,3 +59,6 @@
 #cmakedefine ARROW_WITH_UCX
 
 #cmakedefine GRPCPP_PP_INCLUDE
+#cmakedefine PARQUET_REQUIRE_ENCRYPTION
+
+
```

We may want to group Parquet related definitions in the future:

```suggestion

#cmakedefine PARQUET_REQUIRE_ENCRYPTION
```